BUILD.api

```diff
@@ -40,15 +40,34 @@ gogoslick_proto_library(
     ],
 )
 
+DESCRIPTOR_FILE_GROUP = [
+    "mixer/v1/config/descriptor/attribute_descriptor.proto",
+    "mixer/v1/config/descriptor/label_descriptor.proto",
+    "mixer/v1/config/descriptor/log_entry_descriptor.proto",
+    "mixer/v1/config/descriptor/metric_descriptor.proto",
+    "mixer/v1/config/descriptor/monitored_resource_descriptor.proto",
+    "mixer/v1/config/descriptor/principal_descriptor.proto",
+    "mixer/v1/config/descriptor/quota_descriptor.proto",
+    "mixer/v1/config/descriptor/value_type.proto",
+]
+
 gogoslick_proto_library(
     name = "mixer/v1/config",
     importmap = {
         "google/protobuf/struct.proto": "github.com/gogo/protobuf/types",
+        "mixer/v1/config/descriptor/*.proto": "istio.io/api/mixer/v1/config/descriptor",
```

Comment:
do you want the `*.proto` here? it seems strange to both wildcard it and explicitly list the protos. 

was this meant to be `attribute_descriptor.proto`? Or can we remove it altogether?